<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginForm</class>
 <widget class="QWidget" name="LoginForm">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>483</width>
    <height>607</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">*{
font-size:18px;
background:rgb(163, 96, 240);
}

QLabel
{
font-size: 34px;
font-weight: bold;
}

#errorMessageLabel
{
background:transparent;
color: rgb(255, 0, 0);
font-size: 16px;
}

QFrame
{
background:#333;
border-radius: 15px;
}

QPushButton
{
border: 1px rgb(255, 141, 144);
background:rgb(255, 141, 144);
border-radius:9px;
font-weight: 200;
color: black
}
QPushButton:hover:!pressed
{
background:rgb(204, 114, 117);
border-width:2px;
border-color: white;
border-style:solid;
border-radius:9px;
color: #333;
}

QToolButton
{
 background-color: rgb(255, 141, 144);
 border-width:1px;
 border-radius:200px;
 border-color: red;
 max-width:400px;
 max-height:400px;
 min-width:400px;
 min-height:400px;
}

QLineEdit
{
background:transparent;
border:none;
color:#717072;
border-bottom:1px solid #717072;
}

QTextEdit
{
background:transparent;
border:none;
color:#111;
}</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>110</y>
     <width>381</width>
     <height>381</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="loginFormLabel">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>40</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>BingChillings</string>
    </property>
   </widget>
   <widget class="QPushButton" name="submitPushButton">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>250</y>
      <width>291</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Login</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="usernameLineEdit">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>120</y>
      <width>291</width>
      <height>31</height>
     </rect>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Normal</enum>
    </property>
    <property name="placeholderText">
     <string>Username</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="passwordLineEdit">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>180</y>
      <width>291</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="placeholderText">
     <string>Password</string>
    </property>
   </widget>
   <widget class="QPushButton" name="newUserButton">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>300</y>
      <width>140</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>New User</string>
    </property>
   </widget>
   <widget class="QPushButton" name="guestButton">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>300</y>
      <width>140</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Guest</string>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>-270</y>
      <width>400</width>
      <height>400</height>
     </rect>
    </property>
    <property name="icon">
     <iconset resource="resource.qrc">
      <normaloff>:/images/images/ice-cream.png</normaloff>:/images/images/ice-cream.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="errorMessageLabel">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>530</y>
     <width>381</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Roboto Mono</family>
     <pointsize>-1</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
